deploy/k8s/manifest.yaml

```diff
@@ -73,6 +73,54 @@ data:
               - action: update_label
                 label: status
                 new_label: sw.k8s.node.status
+          - include: k8s.kube_deployment_status_condition
+            experimental_match_labels: { "condition": "Available" }
+            action: insert
+            new_name: k8s.deployment.condition.available
+            operations:
+              - action: filter_datapoints
+                datapoint_value: 1
+                datapoint_value_action: include
+              - action: update_label
+                label: status
+                value_actions:
+                  - value: unknown
+                    new_value: "false"
+              - action: update_label
+                label: status
+                new_label: sw.k8s.deployment.condition.available
```

Comment:
Since we are creating a new metric anyway, this rename is not really necessary. Unless we explicitly want to not use "status" to avoid confusion.